### PR TITLE
Rethinking the README.md

### PR DESCRIPTION
**Pull request by davidortinau**
The readme is heavily focused on contributors. How might we provide a more high level landing page introduction to Xamarin.Forms for both app developers and contributors?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+For quick starts, documentation, and guides read more at https://docs.microsoft.com/en-us/xamarin/xamarin-forms/.
 
 ## Build Status ##
 
 ![Azure DevOps](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/Xamarin/XamarinForms/Xamarin%20Forms?branchName=master "Azure Pipelines")
 
 ## Packages ##
 
-Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [MyGet](https://www.myget.org/F/xamarinforms-ci/api/v2)  (master branch)
+Platform/Feature               | Package name                              | Stable (3.5.0 branch)     |Nightly Feed [MyGet](https://www.myget.org/F/xamarinforms-ci/api/v2)  (master branch)
```
Comment (samhouts) [inline README.md]:
```suggestion
Platform/Feature               | Package name                              | Stable (3.6.0 branch)     |Nightly Feed [MyGet](https://www.myget.org/F/xamarinforms-ci/api/v2)  (master branch)
```

**Comment by samhouts** [comment]
I agree that the readme needs work. I think we do a good job of giving a high level overview of Xamarin.Forms in the documentation, which we do link to from the readme.

**Comment by jsuarezruiz** [comment]
Hey @davidortinau, there's a little conflict here.
I like the idea of adding a link to the "Getting Started" documentation. Would it be interesting to also add links to the API Browser https://docs.microsoft.com/en-us/dotnet/api/?view=xamarin-forms and samples https://github.com/xamarin/xamarin-forms-samples?

**Comment by StephaneDelcroix** [comment]
I'm all for rethinking the README, but this is only small cosmetic changes that do not require discussion. please go ahead with those as soon as you can, @davidortinau

**Pull request closed without merging**